spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -23,18 +23,10 @@
 
     @ems.authentications << FactoryGirl.create(:authentication, cred)
     @ems.update_attributes(:azure_tenant_id => @tenant_id)
-
-    # A true thread may fail the test with VCR
-    allow(Thread).to receive(:new) do |*args, &block|
-      block.call(*args)
-      Class.new do
-        def join; end
-      end.new
-    end
   end
```

Comment:
It was for thread safety for VCR. But VCR added their own safety and keeping it caused problems. We set our own thread limit directly within the armrest gem.